internal/pkg/node/datastructure.go

```diff
@@ -233,3 +239,33 @@ type FileSystemEntry struct {
 
 // string which is printed if no value is set
 const NoValue = "--"
+
+func (e Entry) MarshalText() (buf []byte, err error) {
+	helper := struct {
+		Value  string
+		Source string
+	}{}
+
```

Comment:
The helper here has the effect that every entry will have following output:
```
01:
  AssetKey: ""
  ClusterName: ""
  Comment:
    Source: default
    Value: This profile is automatically included for each node
```
instead of just showing the value. This is a bit like the normal output of the list commands, BUT `json/yaml` is structured data and should as near to `nodes.conf` but with the profiles rendered in. So I think we should ditch this information here. What you think @griznog  and @anderbubble ?
Should we also go for the 4.6.x release with this?

Reply:
I agree that it should match the expected input format for `nodes.conf`.

I would support bumping this to v4.6.0; but I leave it to @griznog to prioritize, as this was his original request.